lib/SendStream.js

```diff
@@ -132,7 +132,7 @@ function SendStream (req, path, options) {
   this._maxage = typeof this._maxage === 'string'
     ? ms.parse(this._maxage)
     : Number(this._maxage)
-  this._maxage = !isNaN(this._maxage)
+  this._maxage = this._maxage === this._maxage // eslint-disable-line no-self-compare
```

Comment:
Can you add a explanation for each changes.
It is not obvious to know what it happen in first view.
Something like `fast path of isNaN(number)`